etna/transforms/feature_selection/filter.py

```diff
@@ -43,6 +40,7 @@ def __init__(
             self.exclude = list(set(exclude))
         else:
             raise ValueError("There should be exactly one option set: include or exclude")
+        self.in_column = "all"
```

Comment:
I thought we wanted to implement more complex logic here.

Reply:
It turned out that all the columns are necessary for this transform